src/runtime/hexagon_host.cpp

```diff
@@ -659,6 +648,35 @@ WEAK const halide_device_interface *halide_hexagon_device_interface() {
     return &hexagon_device_interface;
 }
 
+WEAK void* halide_hexagon_get_support_lib_symbol(void* user_context, const char *name) {
+    /*
```

Comment:
I'd prefer a non-block comment here personally (// instead of /**/)... do we have a convention on this?